### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by uisendy**
In this milestone, creating form input to handle create part of the CRUD operations as performed. 

  -  Added the following forms
       - post form.
       - comment form
  -  Updated routes to create new forms
  -  Updated PostsController.
  -  Added the following controllers
      -  CommentsController.
      -  LikesController

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -4,7 +4,27 @@ def index
     @posts = Post.where(author_id: params[:user_id])
   end
 
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(posts_params)
+    @post.author = current_user
```
Comment (mikethreels) [inline app/controllers/posts_controller.rb]: - [ ] this piece of code should be written like this:
```
@post = @current_user.posts.new(post_params)
```

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.author_id = params[:user_id]
+    @comment.post_id = params[:post_id]
+    @post = Post.find(params[:post_id])
```
Comment (mikethreels) [inline app/controllers/comments_controller.rb]: - [ ] this piece of code should be written like below:
```
    @comment = @current_user.comments.new(params)
    @comment.post_id = params[:post_id]
```

File: app/controllers/likes_controller.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class LikesController < ApplicationController
+  def new
+    @like = Like.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @like = Like.new(post_id: @post.id, author_id: current_user.id)
```
Comment (mikethreels) [inline app/controllers/likes_controller.rb]: - [ ] this line of code should look like:
```
      @like = @current_user.likes.new(post_id: @post.id)
```

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Inieke, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (uisendy) [reply app/controllers/posts_controller.rb]: Thanks for you review. It is very appreciated. I think this is more like an optional suggestion being that the lines of code is more of a suggestion on how to write a one-liner of the same thing i wrote. However, I really appreciate you review and will do as requested. 

Cheers 😄

**Review by danielufeli — APPROVED** [review]
Hi @uisendy,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ChopApproveCopApproveGIF

## Highlights
- Nice commits messages✔️
- Clean Code ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._ate_comments_counter`

**Pull request merged**